container-search/src/main/java/com/yahoo/search/query/properties/QueryProperties.java

```diff
@@ -277,6 +281,9 @@ else if (key.equals(Query.GROUPING_SESSION_CACHE))
                     query.setGroupingSessionCache(asBoolean(value, false));
                 else
                     super.set(key,value,context);
+            } else if (key.toString().equals(GroupingRequest.PROP_REQUEST)) {
```

Comment:
Not needed: Since PROP_REQUEST was private, there is no chance that anybody is already using this property name - it is not part of the API of properties.